datasource-local/src/test/java/cmm/apps/esmorga/datasource_local/event/EventLocalDatasourceImplTest.kt

```diff
@@ -109,4 +112,32 @@ class EventLocalDatasourceImplTest {
         Assert.assertEquals(emptyList<List<EventLocalModel>>(), result)
     }
 
+    @Test
+    fun `given a storage with events when events join event is called then old events are updated with new value`() = runTest {
```

Comment:
I'd say the second "events" word should not be there. And most likely the following ones should be singular.

Reply:
you're right